### PR TITLE
added function which verifies that CRS values are valid

### PR DESCRIPTION
**Pull request by Annie-EXE**
I have added a function which verifies that CRS values are accurate and drops the row otherwise (except for the "cancellation station CRS" column - missing values here do not lead to the row being dropped)

### CODE REVIEW

File: pipeline/transform.py [hunk 1]
```diff
@@ -66,15 +66,62 @@ def create_timestamp_from_date_and_time(df: DataFrame, new_column_name: str,
...
+    rows if the value is not 3 characters long;
+    otherwise, replaces the value with None
+    """
+    df[column_name] = df[column_name].apply(lambda x: str(x).strip().upper()
+                                            if len(str(x).strip()) == 3
+                                            else None)
+
+    if drop_row:
+        df.dropna(subset=[column_name], inplace=True)
+    #     df[column_name] = df[column_name].astype(str)
```
Comment (NgawangT) [inline pipeline/transform.py:93]: commented out code.

File: pipeline/transform.py [hunk 2]
```diff
@@ -66,15 +66,62 @@ def create_timestamp_from_date_and_time(df: DataFrame, new_column_name: str,
```
Comment (NgawangT) [inline pipeline/transform.py:66]: nice lambda functions.

File: pipeline/transform.py [hunk 3]
```diff
@@ -66,15 +66,62 @@ def create_timestamp_from_date_and_time(df: DataFrame, new_column_name: str,
...
+                                            if len(str(x).strip()) == 3
+                                            else None)
+
+    if drop_row:
+        df.dropna(subset=[column_name], inplace=True)
+    #     df[column_name] = df[column_name].astype(str)
+    
+    # else:
+    #     df[column_name] = df[column_name].astype(object)
+    #     df[column_name] = df[column_name].apply(lambda x: None if pd.isna(x) else x)
```
Comment (HarryEP) [inline pipeline/transform.py:97]:
```suggestion
```

**Review by HarryEP — APPROVED** [review]
Nice code! I've made a suggestion of what Ngawang said to make it easier so we can just approve it immediately if you commit the suggestion I think?

File: pipeline/transform.py (hunk 1 above)
Comment (Annie-EXE) [reply pipeline/transform.py]: it shall be purged

**Pull request merged**